Software/src/LedDeviceArdulight.hpp

```diff
@@ -51,6 +51,7 @@ public slots:
 	void updateDeviceSettings();
 	int maxLedsCount(){ return 255; }
 	virtual int defaultLedsCount() { return 25; }
+    void writeLastWill();
```

Comment:
I know formatting is kind of a lost cause on this project but that diff looks horrible while LedDeviceAdalight.hpp  is okay - does that look normal in your IDE?

Reply:
Yeah I try to keep indents consistent, but I jump so much between VS/Xcode/Sublime/Vim so I guess one of them used spaces there... I'll fix it.